whirlpool/types.py

```diff
@@ -11,3 +13,20 @@ class Brand(Enum):
 class Region(Enum):
     EU = 0
     US = 1
+
+
+@dataclass
+class CredentialsDict(TypedDict):
+    client_id: str
+    client_secret: str
```

Comment:
```suggestion
```
this is not used in this PR

Reply:
Fixed in b0f050b0f17b303778c1655d7bf18aa99c322d50